chirp/test/TR_chirp_python.sh

```diff
@@ -9,6 +9,14 @@ c="./hostport.$PPID"
 
 ticket=my.ticket
 
+check_needed()
+{
+	[ -f ../src/python/_CChirp.so ] || return 1
+	python=$(cctools_python -n 3 2.7 2.6 2> /dev/null)
+	[ -n "$python" ] || return 1
+	${python} -c "import json; import Chirp" 2> /dev/null
```

Comment:
One last change, I think remove the redirection to /dev/null. Otherwise it's not possible to know why the test was skipped.